homeassistant/components/energy/validate.py

```diff
@@ -178,27 +178,13 @@ def _async_validate_cost_stat(
             )
         )
 
-
-@callback
-def _async_validate_cost_entity(
```

Comment:
Why wouldn't we just call this from `_async_validate_cost_stat` ? Now `_async_validate_cost_entity` does not validate the entity, as that's all part of `validate_cost_stat`?

Reply:
the entity in this case is auto generated by the energy integration, we only need to check that it's being recorded.